src/UniversalRewardsDistributor.sol

```diff
@@ -122,9 +122,9 @@ contract UniversalRewardsDistributor is IUniversalRewardsDistributor {
             ErrorsLib.INVALID_PROOF_OR_EXPIRED
         );
 
-        amount = claimable - claimed[account][reward];
+        require(claimable > claimed[account][reward], ErrorsLib.CLAIMABLE_TOO_LOW);
```

Comment:
Should the error imply that there's a problem with the `root` instead of implying that the user has used the wrong input value?

Reply:
both of them. We do not need to have a too specific error here. In any case, we'll be able to easily retrieve the exact cause offchain